src/gui/styles/base/basestyle.qss

```diff
@@ -64,3 +64,8 @@ DatabaseWidget #SearchBanner, DatabaseWidget #KeeShareBanner {
     border: 1px solid palette(dark);
     padding: 2px;
 }
+
+QPlainTextEdit, QTextEdit {
+    background-color: palette(base);
+    padding-left: 4px;
+}
```

Comment:
See above.

Reply:
This is also a hack, technically. I looked at the Qt source code for TextEdit and it is a shim on top of a QScrollArea. Qt is entirely unconcerned with alignment styling and box model, even though they profess their support in the docs. It is literally impossible to add interior padding to the QTextEdit, this padding is actually added to the interior of the QScrollArea...